### PR TITLE
Completed javascript-if exercise-Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
Screen Shot 2022-07-07 at 4 06 20 PM
Screen Shot 2022-07-07 at 4 09 35 PM

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (codingcodymiller) [inline javascript-if/main.js:23]: This is backwards. Make sure you test your functions 😉

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person >= 21) {
```
Comment (codingcodymiller) [inline javascript-if/main.js:35]: `person` is meant to be an object, please re-read the instructions for this function.

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person >= 16) {
```
Comment (codingcodymiller) [inline javascript-if/main.js:43]: `person` is meant to be an object, please re-read the instructions for this function.

File: javascript-if/main.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person >= 16) && (person >= 21)) {
```
Comment (codingcodymiller) [inline javascript-if/main.js:51]: `person` is meant to be an object, please re-read the instructions for this function.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 1 above)
Comment (dulceherrera) [reply javascript-if/main.js]: Screen Shot 2022-07-09 at 2 27 29 PM

File: javascript-if/main.js (hunk 4 above)
Comment (dulceherrera) [reply javascript-if/main.js]: Screen Shot 2022-07-09 at 2 34 17 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**